addons/io_scene_gltf2_msfs/__init__.py

```diff
@@ -24,7 +24,7 @@
     "author": "Luca Pierabella, Yasmine Khodja, Wing42, pepperoni505, ronh991, and others",
     "description": "This toolkit prepares your 3D assets to be used for Microsoft Flight Simulator",
     "blender": (3, 3, 0),
-    "version": (1, 3, 3),
+    "version": (1, 3, 3, 1),
```

Comment:
I don't think we should have another sub version

Reply:
ok but thought setting a new version is better to show people there is a newer version to use. Thought 1.3.4 would not show that it's an update to 1.3.3